nixos/modules/services/misc/jackett.nix

```
@@ -58,6 +64,18 @@ in
       description = "Jackett";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
+      preStart = # bash
+        ''
```

```suggestion
      preStart = /* bash */ ''
```
I believe this is the more common format? Or are my habits out of date? At least on my editor this triggers proper nested syntax highlighting.